contracts/MetadataViews.cdc

```diff
@@ -129,4 +129,26 @@ pub contract MetadataViews {
             return "ipfs://".concat(self.cid)
         }
     }
+
+    // Edition is an optional view for collections that issues multiple objects
+    // with the same name, description and media file. This information is 
+    // useful for wallets and marketplaces.
+    // 
+    pub struct Edition {
+
+        // The edition number of the object.
+        //
+        pub let editionNumber: UInt64
+
+        // The max edition number of this type of objects.
+        // 
+        // This field should only be provided for limited-editioned objects.
+        // 
+        pub let maxEdition: UInt64?
```

Comment:
Can you add a little bit more detail in the comments about what edition number and maxEdition mean? I assume edition number is like a "24 of 100" type thing and maxEdition is 100 in that case, but that should be crystal clear from the comments and docs here

Reply:
updated.